spec/default_spec.rb

```diff
@@ -62,14 +62,14 @@
   end
 
   it 'expects nrpe config to have allow_bash_command_substitution when set' do
-    chef_run.node.set['nrpe']['allow_bash_command_substitution'] = 0
```

Comment:
This is a bug in the older cookstyle. Upgrade workstation / DK and this will fix it. This should autocorrect to ```chef_run.node.normal```

Reply:
Huh interesting, I thought I installed the latest brew